storage/persistent_storage/persistent-storage-iscsi.adoc

```diff
@@ -31,10 +31,13 @@ By default, they are ports `860` and `3260`.
 
 [IMPORTANT]
 ====
-OpenShift assumes that all nodes in the cluster have already configured the iSCSI
-initiator, that is, users have installed the `iscsi-initiator-utils` package and configured
-their initiator name in `/etc/iscsi/initiatorname.iscsi`. See Storage
-Administration Guide linked above.
+Users must ensure that the iSCSI initiator is already configured on all 
+{product-title} nodes, that is, users must have installed the `iscsi-initiator-utils` 
```

Comment:
```suggestion
{product-title} nodes, meaning that users must have installed the `iscsi-initiator-utils` 
```
This update replaces ',that' which is a restrictive clause with no comma. Something similar to my suggestion will also work.